### ISSUE TITLE
Errors when i run http://doubleclick.net/?debug=conf

### ISSUE THREAD

**Issue by frankied210** [WaLLy3K/Pi-hole-Block-Page#23]
I'm getting a slew of  errors like the one below when I brows to doubleclick.net/?debug=conf

> Warning:  array_merge(): Argument #1 is not an array in /var/www/html/index.php on line 174

I've upload the entire output of the degug=conf here https://s3.amazonaws.com/pi-hole-files/doubleclickDOTnetDebugConf.txt

I've also uploaded the index.php file here https://s3.amazonaws.com/pi-hole-files/index.php

I tried to a uninstall and reinstall but it sill isn't working like it should. I get the block page but it doesn't give me the classifcatrion for the block.

I've also uploaded the php.ini file here https://s3.amazonaws.com/pi-hole-files/phbp.ini

Any help would be much appreciated.
Also, I'd like to add Pornography classification, assistance with getting that added would also be appreciated. 

 I'm running  Pi-hole Version v3.1.4  Web Interface Version v3.1  FTL Version v2.9.4

**Comment by WaLLy3K** [comment]
That's odd! Are you aware of the exact PHP version you're using? Alternatively, run `pihole -d` and provide the debug token you get when the log is uploaded.

**Comment by frankied210** [comment]
Pi-hole debug token is mg4ro3pyb9

PHP Version info: PHP 5.6.30-0+deb8u1 (cli) (built: Apr 14 2017 16:20:58)

**Comment by WaLLy3K** [comment]
Thanks for that!

From what I can tell, it seems there's an issue with `/var/www/html/classification.ini` (PHBP debug has `PHBP info: v2.3.2 (RCI: v, 0a6e236c)` instead of `RCI: v2.32`). Try deleting it, and re-visiting http://doubleclick.net/?debug=conf

**Comment by frankied210** [comment]
So to be clear, you want me to delete classification.ini?  I can try tomorrow and post my results.

**Comment by frankied210** [comment]
I removed the classification.ini and it seems to be working now.  Should it be showing multiple classifications if a url is on multiple lists in different classifications?

**Comment by WaLLy3K** [comment]
>  Should it be showing multiple classifications if a url is on multiple lists in different classifications?

No, it should only show the most significant classification - so if a site appeared in a site defined as Malware as well as Suspicious, it'll be flagged as Malware.

(Apologies for the delay in reply!)
